source/reference/minio-server/settings/console.rst

```diff
@@ -0,0 +1,119 @@
+.. _minio-server-envvar-console:
+
+====================================================
+Environment Variables to Configure the MinIO Console
+====================================================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+The following environment variables control behavior for the embedded MinIO Console:
```

Comment:
```suggestion
The following environment variables control behavior for the embedded MinIO Console.
There are no equivalent settings to configure via :mc:`mc admin config set`.
```

Reply:
Trying a different route. Using tabs and mentioning that the setting does not have a config option. This lines up with what I do elsewhere, though none of the settings on this particular page have a config option.